src/naquibalam/A12_3_Scanner.java

```diff
@@ -0,0 +1,70 @@
+package naquibalam;
+
+import java.util.Scanner;
+
+public class A12_3_Scanner {
+
+	char getFirstNonRepeatingChar(String input) {
+		char returnChar = 0;
+		for (int i = 0; i <= input.length() - 1; i++) {
+			boolean repeatingFound = false;
+			for (int j = 0; j <= input.length() - 1; j++) {
+				if ((input.charAt(i) == input.charAt(j)) && (i != j)) {
+					repeatingFound = true;				
+				} 
+			}
+			if (repeatingFound == false) {
+				returnChar = input.charAt(i);
+				break;
+			}
+		}
+		return returnChar;
+	}
+	
+	char getFirstRepeatingChar(String input) {
+		char returnChar = 0;
+		for (int i = 0; i <= input.length() - 1; i++) {
+			boolean repeatingFound = false;
+			for (int j = 0; j <= input.length() - 1; j++) {
```

Comment:
here you can also use 
indexOf(ch) != lastIndexOf(ch) logic to avoid one more for loop